tests/pyrog/migrations/test_0009_move_column_fk_on_input.py

```diff
@@ -29,11 +29,17 @@ def test_migrate(migrator, state, owner_factory, input_group_factory):
 
 
 @pytest.mark.migration(app_label="pyrog", migration_name="0009_move_column_fk_on_input")
-def test_migrate_reverse(migrator, state, input_factory, column_factory):
-    input = input_factory()
-    column = column_factory()
+def test_migrate_reverse(migrator, state, input_group_factory, column_factory):
+    InputGroup = state.apps.get_model("pyrog", "InputGroup")
+    input_group_factory()
```

Comment:
maybe we shouldn't use factory in migration tests since they refers to the current model. if we apply further changes, this test will need an update.

Reply:
You're right!